ddtrace/contrib/langchain/patch.py

```diff
@@ -629,9 +632,12 @@ async def traced_chain_acall(langchain, pin, func, instance, args, kwargs):
     span = integration.trace(pin, "%s.%s" % (instance.__module__, instance.__class__.__name__), interface_type="chain")
     final_outputs = {}
     try:
-        inputs = get_argument_value(args, kwargs, 0, "inputs")
-        if not isinstance(inputs, dict):
-            inputs = {instance.input_keys[0]: inputs}
+        if SHOULD_PATCH_LANGCHAIN_COMMUNITY:
+            inputs = get_argument_value(args, kwargs, 0, "input")
+        else:
+            inputs = get_argument_value(args, kwargs, 0, "inputs")
+            if not isinstance(inputs, dict):
```

Comment:
should this `if` apply in both cases? or just when it's an older langchain version (seems in the non async traced method, it's done in either case)?

Reply:
Yeah good catch, I missed this